letsdrill.rb

```diff
@@ -1,24 +1,37 @@
+#letsdrill - Rajashree
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  if integer >= 90 && integer <=100
```

Comment:
Check out something called a guard clause. It will shorten this method tremendously. In addition, you might want to check out other ways of doing this in a more ruby way with ranges.